adapters/adxcg/adxcg.go

```diff
@@ -0,0 +1,130 @@
+package adxcg
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v14/openrtb2"
+	//"github.com/mxmCherry/openrtb"
```

Comment:
Please remove the commented out import package. Please also be aware there is a PR to upgrade `mxmCherry/openrtb` to version 15 which will eventually be a conflict to be resolved. Should be simple, it's the import path.

Reply:
done

Reply:
That was a very quick response. Perhaps you'll make it in before the `mxmCherry/openrtb`.